src/locales/en.json

```diff
@@ -55,7 +55,8 @@
     "selectLanguage": "Select Language",
     "scrollBackToContent": "Scroll back to content",
     "zoomIn": "Zoom in",
-    "zoomOut": "Zoom out"
+    "zoomOut": "Zoom out",
+    "resetZoom": "Reset zoom"
```

Comment:
`zoomReset` maybe?

Reply:
I thought to add it like this as well but wanted to keep the conventions that was set before me, and the component name and action is resetZoom.

Reply:
@lipis do you want to change it?

Reply:
It's alright.. at least it's consistent with something :)